.env.template

```diff
@@ -4,7 +4,10 @@ FLASK_ENV=development
 
 # Change the following values for local development.
 SECRET_KEY=secret-key
-TRELLO_API_KEY=
-TRELLO_TOKEN=
-TRELLO_BOARD_ID=
-TRELLO_BASE_URL= 'https://api.trello.com/1'
\ No newline at end of file
+COSMOS_CONNECTION_STRING=cosmos_connection_string
+DATABASE=database
+CLIENT_ID=
+CLIENT_SECRET=
+GET_USER_IDENTITY_URL="https://github.com/login/oauth"
+GET_ACCESS_TOKEN_URL="https://github.com/login/oauth/access_token"
+GET_USER_URL="https://api.github.com/user"
```

Comment:
Although parameterising these feels like it _might_ be useful (if GitHub changed one in a hurry) it's potentially suggesting the system is more adaptable than it really is; although the oauth endpoints should be following standard patterns, the `user` one won't necessarily be and so switching to another provider is very unlikely to be achievable with just these. Not a major issue, but personally I'd lean towards specifying these directly in the code rather than here